test/cloud/digital_ocean_test.rb

```diff
@@ -8,13 +8,21 @@ class DigitalOceanTest < Test::Unit::TestCase
 
     setup do
       env = {
-        'client_key' => "XXX",
-        'api_key' => "YYY",
-        'region' => 'New York 1',
-        'key_file' => "#{File.dirname(__FILE__)}/../fixtures/basic/test.pem"
+        'digital_ocean_token' => "XYZ",
+        'region' => 'nyc1',
+        'key_file' => "#{File.dirname(__FILE__)}/../fixtures/basic/test.pem",
+        'key_name' => 'test'
       }
       env = Rubber::Configuration::Environment::BoundEnv.new(env, nil, nil, nil)
       @cloud = Rubber::Cloud::DigitalOcean.new(env, nil)
+
+      @cloud.compute_provider.ssh_keys.each do |key|
+        @cloud.compute_provider.delete_ssh_key(key.id)
+      end
+
+      # This is currently (as of 11/2/15) the only valid image name in
+      # Fog's mocked digital ocean images request
+      @valid_image_name = "Nifty New Snapshot"
```

Comment:
Minor style nitpick, but we generally use single-quoted strings if interpolation isn't being used.